packages/google_maps_flutter/google_maps_flutter_ios/pigeons/messages.dart

```diff
@@ -546,6 +561,25 @@ enum PlatformMapBitmapScaling {
   none,
 }
 
+class PlatformBitmapPinConfig {
```

Comment:
This should have a comment like the others indicating what other class it corresponds to.